src/test/java/org/opensearch/knn/index/codec/KNNCodecTestCase.java

```diff
@@ -109,7 +109,8 @@ public void testFooter(Codec codec) throws Exception {
         doc.add(vectorField);
         writer.addDocument(doc);
 
-        NativeMemoryLoadStrategy.IndexLoadStrategy.initialize(createDisabledResourceWatcherService());
+        ResourceWatcherService resourceWatcherService = createDisabledResourceWatcherService();
+        NativeMemoryLoadStrategy.IndexLoadStrategy.initialize(resourceWatcherService);
```

Comment:
It seems that the only change in these lines is related to improved code readability, is this correct?

Reply:
I did it to be able to close resourceWatcherService at the end.

Reply:
Oh, I see, that makes sense. Thank you Jack